vizro-core/docs/pages/explanation/contributing.md

```diff
@@ -1,192 +1,159 @@
 
-## Found a bug?
+There are two ways to develop on Vizro:
 
-Head over to our [issues page](https://github.com/mckinsey/vizro/issues) and raise a ticket in the category `bug report`. It would help us if you could first check if there are any existing issues with a similar description before submitting a new ticket. We will try to reproduce the bug you've reported and follow up with the next steps.
+* [GitHub Codespaces](https://docs.github.com/en/codespaces). This is the recommended method if you are a new contributor and is the quickest and easiest way to get started. All development can be done in your browser in a temporary environment; you do not need to set up anything on your computer. Read [Develop on GitHub Codespaces](#develop-on-github-codespaces) for full instructions on how to do this.
+* Local machine. If you are more experienced then you might prefer to develop on your own computer. Read [Develop locally](#develop-locally) for full instructions on how to do this.
 
-## Want to request a feature?
+!!! note
 
-To raise a feature request, head to our [issues page](https://github.com/mckinsey/vizro/issues) and raise a ticket in the category `feature request`. We would appreciate if you searched the existing issues for a similar description before raising a new ticket. The team will then try to understand the request in more detail, explore the feasibility and prioritize it in relation to the current roadmap. We will get back to you as soon as possible with an estimate of whether and when this feature could be released.
+    For either method, Hatch is the _only development dependency_. You do not need to manually install Python or create any virtual environments to develop Vizro; all this will be handled for you behind the scenes by Hatch. We have also configured our codespace to pre-install Hatch. If you develop on GitHub Codespaces you don't need to install anything at all!
 
-## Got a Vizro question?
 
-We are happy to receive general questions around Vizro. Take a look at our [issues page](https://github.com/mckinsey/vizro/issues) and raise a ticket in the category `general question`. We would be grateful if you could check for any similar descriptions in the existing issues before opening a new ticket.
+## Develop on GitHub Codespaces
 
-If you are already active on the Plotly Community Forum, you can also ask your question over there in the [Dash Python category](https://community.plotly.com/c/python/25).
+!!! note
 
-## How to interact with the repository
+    There is no need to manually create a fork of the Vizro code if you use GitHub Codespaces, because fork is [automatically created for you](https://docs.github.com/en/codespaces/developing-in-a-codespace/using-source-control-in-your-codespace#about-automatic-forking).
```

Comment:
```suggestion
    There is no need to manually create a fork of the Vizro code if you use GitHub Codespaces, because a fork is [automatically created for you](https://docs.github.com/en/codespaces/developing-in-a-codespace/using-source-control-in-your-codespace#about-automatic-forking).
```

As I said in catchup - this actually was not 100% without friction. It did work, but there was an error message.

Reply:
Indeed... I would be tempted to leave these instructions as they are here (assuming the automatic fork works) and then you show the other flow where you do a manual fork in your GHC. wdyt?

Reply:
Hm ok - let's do that. Do you then think it is an issue that I even show something different at GHC - I just don't want people to even encounter any error messages

Reply:
As mentioned above, I am now not creating a fork first for GHC, but let's be aware that there sometimes can be issues..